keyboards/durgod_k320/readme.md

```diff
@@ -0,0 +1,25 @@
+# durgod_k320
+
+This is a standard fixed ANSI TKL from off the shelf Durgod Taurus K320 without Backlight.
+
+* Keyboard Maintainer: [kuenhlee](https://github.com/kuenhlee)
+* Hardware Supported: Durgod Taurus K320 board with STM32F070RBT6
+* Hardware Availability: https://www.durgod.com/page9?product_id=47&_l=en
+                       : 
+
+## Instructions
+
+### Build
+
+Make example for this keyboard (after setting up your build environment):
+
+    make durgod_k320:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+### Flash using ST's STSW-STM32080 : https://www.st.com/en/development-tools/stsw-stm32080.html
```

Comment:
Looks like the F070 uses the same USB DFU bootloader as the F072/F303/etc. so if you change the driver from STTub30 to WinUSB, you should be able to flash using dfu-util as normal (and this will allow you to use the Toolbox or `qmk flash`).

Reply:
I am only 2 days old with QMK, let me to how to get this done and test it on my board. Its kind of pain now to electrically short the pins each time to flash the board for testing.

Reply:
@fauxpark I see what you meant now. You were actually saying the Bootloader's USB driver on my Windows and not the Keyboard Firmware to use different USB stack. Its a bit troublesome for me if to switch to WinUSB as my work's laptop needed to use STTub30 and switching back and forth will be a headache for me.